README.md

```diff
@@ -1,8 +1,61 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Khanjan Dabhi . Jarvis Consulting
 
+Driven data science aficionado familiar with gathering, cleaning, and organizing data for use by technical and non-technical personnel. Progressed understanding of measurable, mathematical, and analytical procedures. Highly organized, motivated, and diligent with significant background in software engineering. Computer vision enthusiast. Ready to thrive in demanding digital intelligence processing environments. Well-informed on latest machine learning advancements. Ready to combine tireless hunger for new skills with desire to exploit cutting-edge data science technology.
```

Comment:
Driven data science aficionado familiar with the gathering, cleaning, and organizing data for use by technical and non-technical personnel. Progressed understanding of measurable, mathematical, and analytical procedures. Highly organized, motivated, and diligent with significant background in software engineering. Computer vision enthusiast. Ready to thrive in demanding digital intelligence processing environments. Well-informed on the latest software and machine learning advancements. Ready to combine tireless hunger for new skills with a desire to exploit cutting-edge data science technology.

focus on software development more..